core/livepeernode.go

```diff
@@ -94,25 +94,13 @@ func NewLivepeerNode(e eth.LivepeerEthClient, wd string, dbh *common.DB) (*Livep
 
 func (n *LivepeerNode) StartEthServices() error {
 	var err error
-	for k, s := range n.EthServices {
-		// Skip BlockService until the end
-		if k == "BlockService" {
-			continue
-		}
+	for _, s := range n.EthServices {
 		err = s.Start(context.Background())
```

Comment:
What service are we starting here? aren't we removing the existing EthServices completely?

Reply:
We want to eventually remove the existing `EthServices` completely once we finish refactoring the existing services. I was planning on refactoring each existing service in its own PR and then once all of them are refactored completely remove `EthServices`.

Reply:
Oh yeah that sounds very reasonable !